ui/app/AppLayouts/Profile/Sections/BackupSeedModal.qml

```diff
@@ -0,0 +1,117 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import "../../../../imports"
+import "../../../../shared"
+
+ModalPopup {
+    id: popup
+
+    title: qsTr("Write down your seed phrase")
+
+    Item {
+
+      id: seed
+      anchors.left: parent.left
+      anchors.right: parent.right
+      width: parent.width
+      height: children[0].height
+
+      Component {
+        id: wordCmp
+
+        Item {
+          height: 40
+          anchors.left: parent.left
+          anchors.leftMargin: Theme.bigPadding
+
+          TextEdit {
+            text: word
+            font.pixelSize: 15
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: Theme.smallPadding
+            selectByMouse: true
+            readOnly: true
+          }
+        }
+      }
+
+
+      Rectangle {
+        anchors.top: parent.top
+        anchors.topMargin: Theme.padding
+        height: 250
```

Comment:
This is actually a problem. Mnemonics can be 6, 12, 24 words, meaning that the height of this Rectangle (and its ListView effectively) need to be dynamic.

I tried making this work with `children` property and some calculation but it didn't work for some reason.

Also, I believe the modal itself has a fixed height at the moment, so that needs to be tackled as well at some point.

Reply:
If you handle the list in QML, you can do `mnemonic.split(' ').length * heightOfOneThingy`
You can also set a custom height to the modal using `customHeight`